examples/pie_and_polar_charts/nested_pie.py

```diff
@@ -17,17 +17,30 @@
 #
 # In this case, pie takes values corresponding to counts in a group.
 # We'll first generate some fake data, corresponding to three groups.
-# In the outer circle, we'll treat each number as belonging to its
-# own group. In the inner circle, we'll plot them as members of their
+# In the inner circle, we'll treat each number as belonging to its
+# own group. In the outer circle, we'll plot them as members of their
 # original 3 groups.
+#
+# The effect of the donut shape is achieved by setting a `width` to
+# the pie's wedges through the `wedgeprops` argument.
+
 
-vals = np.array([[1, 2, 3, 4], [2, 3, 4, 5], [3, 4, 5, 6]])
 fig, ax = plt.subplots()
-ax.pie(vals.flatten(), radius=1.2,
-       colors=plt.rcParams["axes.prop_cycle"].by_key()["color"][:vals.shape[1]])
-ax.pie(vals.sum(axis=1), radius=1)
-ax.set(aspect="equal", title='Pie plot with `ax.pie`')
 
+size = 0.3
+vals = np.array([[60., 32.], [37., 40.], [29., 10.]])
+
+cm = plt.get_cmap("tab20c")
+cout = cm(np.arange(3)*4)
+cin = cm(np.array([1, 2, 5, 6, 9, 10]))
```

Comment:
Can you please make the variables more telling? i.e. `outer_colors`, `inner_colors`, `cmap`.

Reply:
Agreed.